.build/perform-release.sh

```diff
@@ -19,7 +19,7 @@ cd release-work
 echo -e "${BLUE}Switching to tag${NC}"
 git checkout ${LAST_TAG}
 echo -e "${BLUE}Running deployment${NC}"
-mvn deploy -Psonatype,release
+mvn deploy -DwithStaging -DwithSign
```

Comment:
Any reason to change this?

Reply:
Well, the reason is the flexibility provided by property activation. We can run the building locally and combine activation of multiple profiles using only one property, for example.